coordinator/src/position/models.rs

```diff
@@ -172,10 +173,10 @@ fn calculate_accept_settlement_amount(
         Direction::Short => short_leverage,
     };
 
-    let margin_trader_without_opening_fees = calculate_margin(opening_price, quantity, leverage);
+    let margin_trader = calculate_margin(opening_price, quantity, leverage);
 
-    let accept_settlement_amount = Decimal::from(margin_trader_without_opening_fees)
-        + Decimal::from(pnl)
+    let accept_settlement_amount = Decimal::from(margin_trader) + Decimal::from(pnl)
+        - Decimal::from(open_position_fee)
```

Comment:
🔧
Note that with this approach the confirmation screen in the app is not correct: 

<img width="374" alt="image" src="https://github.com/get10101/10101/assets/224613/a52d418b-776f-4d57-ae08-80a786875b64">

The fee will be deducted from the `margin`+`pnl`... so the `Total` amount is wrong as the user only has to provide `margin` initially`.

Reply:
Hmm.. that's true. Should we keep the other approach then?

Reply:
I guess we should get the old "invoice-approach" again in the meantime.